docker/mysql-setup/init.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 
 sed -e "s/DATAHUB_DB_NAME/${DATAHUB_DB_NAME}/g" /init.sql | tee -a /tmp/init-final.sql
-mysql -u $MYSQL_USERNAME -p"$MYSQL_PASSWORD" -h $MYSQL_HOST < /tmp/init-final.sql
\ No newline at end of file
+(($MYSQL_PORT)) ||  port="3306"
```

Comment:
I am not sure if this line is required or not. In the helm charts, the port param is defined and passed to this task.
See [definition](https://github.com/acryldata/datahub-helm/blob/fa53c6d15963760315eeedd3067bc29551cfb92d/charts/datahub/values.yaml#L134)
See [the env var passed to this job](https://github.com/acryldata/datahub-helm/blob/fa53c6d15963760315eeedd3067bc29551cfb92d/charts/datahub/templates/mysql-setup-job.yml#L60)

Still, I couldn't be sure about the other deployment flows of datahub other then using helm. Also considering that [the port param was not mentioned in the docs](https://github.com/acryldata/datahub-helm/pull/125) too, I thought it might be better to add a default value to here would be better. WDYT?

Reply:
Can you try switching to bash by changing line 1 to #!/bin/bash
and then use this syntax to set defaults? https://github.com/datahub-project/datahub/blob/master/docker/kafka-setup/kafka-setup.sh#L2

Reply:
Nice suggestion, did it your way @dexter-mh-lee !
Could you take a look once again please?